docker/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN cargo build --locked --profile $PROFILE $BUILD_ARGS
 # stage 2: packaging
 # ==========================
 FROM ubuntu:20.04
-LABEL maintainer="Litentry Technologies GmbH <info@litentry.com>"
+LABEL maintainer="Trust Computing GmbH <info@litentry.com>"
```

Comment:
What about the email? (there're more occurrences)

Reply:
Confirmed it with Hanwen, the email is info@litentry.com。